src/components/PersonalInformation/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useContext } from "react";
+import DateInput from "../DateInput/index.jsx";
+import TextInput from "../TextInput/index.jsx";
+import styles from "./styles/styles.module.scss";
+import SurveyInformation from "../SurveyInformation/index.jsx";
+
+let isValid = [false, false, false];
+
+const PersonalInformation = () => {
```

Comment:
This component needs a bit of refactor. But if it works, lets move on and please make sure that we can setup a time to go through the structure and improve it. Should be the latest notion card if I'm not mistaken.

Reply:
thanks